maths/find_max.rb

```diff
@@ -0,0 +1,66 @@
+# A ruby program to find max from a set of elements
+
+module FindMax
+  # This find_max method will return the max element out of the array
+  def self.find_max(*array)
```

Comment:
I've noticed a pattern in this repo of having each approach defined in a separate method rather than all within the same module. Perhaps we should do this to stay consistent?